Makefile.pre.in

```diff
@@ -1427,7 +1427,8 @@ regen-opcode:
 		$(srcdir)/Lib/opcode.py \
 		$(srcdir)/Include/opcode.h.new \
 		$(srcdir)/Include/internal/pycore_opcode.h.new \
-		$(srcdir)/Include/internal/pycore_intrinsics.h.new
+		$(srcdir)/Include/internal/pycore_intrinsics.h.new \
+		$(srcdir)/Lib/_opcode_metadata.py
```

Comment:
Assuming this is an input file, why not move it to the second position, after opcode.py?